### DISCUSSION TITLE
An other console extension

### DISCUSSION THREAD

**Discussion by OeildeLynx31** [TurboWarp/extensions#237] [Show and tell]
I made an other console extension, and it is more complete, because it has more blocks, with more interesting function, like custom background  color, or size on the console.
It doesn't need to be run unsanboxed, it is a sandboxed extension.
You can find the source code [here](https://github.com/OeildeLynx31/Console-extension/blob/main/extension-console.js), and use the extension in TurboWarp with [this link](https://turbowarp.org/editor?extension=https://oeildelynx.top7box.com/fichiers/extension/console.js).

What do you think about it?
_(Sorry if there are english errors, I am not english)_

**Comment by SIPC** [comment]
I think the color is not necessary, but group I will add

**Reply by SIPC** [reply]
So thank you for your advice
